client/packages/invoices/src/Prescriptions/DetailView/History/columns.ts

```diff
@@ -1,30 +1,41 @@
-import { ColumnDescription, useColumns } from '@openmsupply-client/common';
+import { ColumnDef, useTranslation } from '@openmsupply-client/common';
 import { HistoryItem } from './HistoryModal';
+import { useMemo } from 'react';
 
 export const useHistoryColumns = () => {
-  const columns: ColumnDescription<HistoryItem>[] = [
-    {
-      label: 'report.item-name',
-      key: 'itemName',
-    },
-    {
-      label: 'label.unit-quantity',
-      key: 'unitQuantity',
-    },
-    {
-      label: 'label.directions',
-      key: 'directions',
-    },
-    {
-      label: 'label.date',
-      key: 'date',
-      accessor: ({ rowData }) => rowData.date?.toLocaleDateString(),
-    },
-    {
-      label: 'label.prescriber',
-      key: 'prescriber',
-    },
-  ];
+  const t = useTranslation();
 
-  return useColumns(columns, {}, []);
+  const columns = useMemo(
+    (): ColumnDef<HistoryItem>[] => [
+      {
+        accessorKey: 'itemName',
+        header: t('report.item-name'),
+        size: 150,
+      },
+      {
+        accessorKey: 'unitQuantity',
```

Comment:
This column should probably use `columnType: ColumnType.Number` to round off this precision issue:

<img width="525" height="164" alt="Image" src="https://github.com/user-attachments/assets/762c7961-773f-4bf0-b7b1-fbcafc7bd589" />

Or maybe check how "Quantity" columns are rendered in other tables and copy that?